### PR TITLE
✨ Add mint nft script

### PR DESCRIPTION
**Pull request by williamchong**
TODO: 
- [x] parse cmd argument
- [x] parse uri to inject nft class/index as qs
- [ ] allow setting id per nft

### CODE REVIEW

File: mint-nft/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+npm install
+```
+
+## Configuration
+
+Change the data in `./list.csv` (reference `./list_example.csv`)
+
+Change the data in `./config/config.js`
+
+Where `MNEMONIC` is your mnemonic phrase , `WAIT_TIME` is wait time after console (before send)
```
Comment (nnkken) [inline mint-nft/README.md:19]: Seems not the case? And it seems there is no config requiring customization in `config.js`

**Review by nnkken — APPROVED** [review]
Everything other than README are OK

File: mint-nft/README.md (hunk 1 above)
Comment (williamchong) [reply mint-nft/README.md]: updated

File: mint-nft/index.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+  } = args;
+  if (help) {
+    printHelp();
+    return;
+  }
+  if (!nftCount || !Number(nftCount)) {
+    console.error('Invalid NFT count');
+    printHelp();
+    return;
+  }
```
Comment (WeiJun0827) [inline mint-nft/index.js:151]: Do we have to check the limitation on `nftCoint`?

Comment (williamchong) [reply mint-nft/index.js:151]: maybe we can check mintCount <= nftCount

**Pull request merged**